activerecord/lib/active_record/relation/delegation.rb

```diff
@@ -89,6 +89,8 @@ def method_missing(method, *args, &block)
             self.class.delegate_to_scoped_klass(method)
             scoping { @klass.public_send(method, *args, &block) }
           elsif arel.respond_to?(method)
+            ActiveSupport::Deprecation.warn \
+              "Delegating #{method} to arel is deprecated and will be removed in Rails 6.0."
```

Comment:
I removed `"Use relation.arel.#{method} instead."` in the message because it's private API and fixed the next version of Rails to 6.0.